api/v1beta2/network_types.go

```diff
@@ -36,6 +36,9 @@ type NetworkStatus struct {
 
 	// APIServerELB is the Kubernetes api server load balancer.
 	APIServerELB LoadBalancer `json:"apiServerElb,omitempty"`
+
+	// NatGatewaysIPs contains the NAT Gateways public IPs.
+	NatGatewaysIPs []string `json:"natGatewaysIPs,omitempty"`
```

Comment:
```suggestion
	// NatGatewaysIPs contains the public IPs of the NAT Gateways
	NatGatewaysIPs []string `json:"natGatewaysIPs,omitempty"`
```

This variable is named plural-plural on purpose and we should make that clear. Using an apostrophe like "the NAT Gateways' public IPs" probably isn't clear enough, so my suggestion rewords it in another way.

It's different from what we have with

```
InstanceIDs
SecurityGroupIDs
SubnetIDs
SourceSecurityGroupIDs
PolicyARNs
```

because this collects the IP for each of the (typically) 3 NAT gateways, each gateway normally having one IP. So it shouldn't be named `NatGatewayIPs`.